### ISSUE TITLE
Does it automatically add ads.txt files?

### ISSUE THREAD

**Issue by efuo** [nuxt-modules/google-adsense#197]
Does it automatically add ads.txt files?

**Comment by manniL** [comment]
No, it does not. Seems like a good idea for a PR though (generation enabled by default but can be disabled)

**Comment by efuo** [comment]
👌, I'll PR

**Comment by efuo** [comment]
This file can be written directly to the public directory. The feature doesn't seem to be important.

**Comment by manniL** [comment]
@efuo yes it can be, but why not add QOL to do so by the module? 😋

**Comment by efuo** [comment]
> @efuo yes it can be, but why not add QOL to do so by the module? 😋

you're right.🤣
